src/Avalonia.Controls/Presenters/TextPresenter.cs

```diff
@@ -292,6 +292,8 @@ protected void InvalidateFormattedText()
                 _constraint = _formattedText.Constraint;
                 _formattedText = null;
             }
+
+            InvalidateVisual();
```

Comment:
Given that we're now manually calling `InvalidateVisual` here, is the change from https://github.com/AvaloniaUI/Avalonia/pull/3643/files still necessary?

Reply:
Yes because InvalidateVisual just tells the renderer to re-evaluate the FormattedText instance but without reacting to TextProperty.Changed the FormattedText will never be recreated with the updated text.

Reply:
We might want to compare all values that affect the FormattedText instance and recreate it on render when something has changed. That way we can just use `AffectsRender<>`

Reply:
Ok, lets leave this as it is for now.